ci/check-documentation.py

```diff
@@ -29,22 +30,41 @@
     'tt', 'u', 'ul', 'var',
 ])
 
-def find_md_files():
-    thirdparty_dir = OMZ_ROOT / 'demos' / 'thirdparty'
+thirdparty_dir = OMZ_ROOT / 'demos' / 'thirdparty'
+
 
+def find_md_files():
     for path in OMZ_ROOT.glob('**/*.md'):
         if thirdparty_dir in path.parents: continue
         yield path
 
 def main():
     all_passed = True
+    
+    md_check_cases = (f'{OMZ_ROOT}{os.sep}models{os.sep}intel{os.sep}index.md', 
+                     f'{OMZ_ROOT}{os.sep}models{os.sep}public{os.sep}index.md', 
+                     f'{OMZ_ROOT}{os.sep}demos{os.sep}README.md')
 
     def complain(message):
         nonlocal all_passed
         all_passed = False
         print(message, file=sys.stderr)
 
     for md_path in sorted(find_md_files()):
+        check_md_links = False
+        readme_files = list()
+
+        if os.path.normpath(md_path) in md_check_cases:
+            check_md_links = True
+            path_folder = os.path.dirname(md_path)
+
+            md_paths = tuple(Path(path_folder).glob('*/*/README.md')) + \
+                       tuple(Path(path_folder).glob('*/README.md'))
+
+            # transforming paths to url format
```

Comment:
I don't think that's a good idea. Comparing URLs is generally difficult (i.e. `README.md` is equivalent to `README%2Emd`), and it's not necessary here. Just keep the paths as paths, and convert the URL to path instead. (In fact, you shouldn't even need to do _that_, as the script already does it for link checking.)